docker/tds/docker-compose.yml

```diff
@@ -178,3 +180,14 @@ services:
     environment:
       - RABBITMQ_LOGS="-"
       - RABBITMQ_SASL_LOGS="-"
+
+  #Redis data store
+  redis:
```

Comment:
Redis caching will be enabled for the local-docker profile.  Enabling redis for the local-development profile works for me.